### DISCUSSION TITLE
Redshift compatibility

### DISCUSSION THREAD

**Discussion by CTW01218** [brooklyn-data/dbt_artifacts#432] [Q&A]
Since AWS Redshift is somehow based on Postgres, is it possible to use the package in Redshift?

**Comment by hanslemm** [comment]
[PR](https://github.com/brooklyn-data/dbt_artifacts/pull/453) is here, you can use in your dbt project

**Reply by darekdade** [reply]
@hanslemm Thanks for this! Any idea if this PR will get approved anytime soon?

**Reply by hanslemm** [reply]
I have no hopes as no one answered my PR @darekdade
